tests/testnet/voting.ts

```diff
@@ -0,0 +1,94 @@
+import { expect } from 'chai'
+import { CandidateName, Voting, N } from '../../src/contracts/voting'
+import {
+    getDefaultSigner,
+    getRandomInt,
+    sleep,
+    stringify,
+} from '../utils/helper'
+import { FixedArray, MethodCallOptions, toByteString, Scrypt } from 'scrypt-ts'
+
+async function main() {
+    Scrypt.init({
+        apiKey: 'alpha_test_api_key',
+        baseUrl:
+            'https://dev-platform-service-testnet-6q7ze.ondigitalocean.app',
```

Comment:
- Hide it from user.
- 
- Add a switch for testnet/mainnet

Reply:
we can hide it when connect mainnet:

```ts
    Scrypt.init({
        apiKey: 'alpha_test_api_key',
    })
```

but if connect testnet  or localhost, we need it:

```ts
    Scrypt.init({
        apiKey: 'alpha_test_api_key',
        baseUrl: 'https://testnet.api.scrypt.io',
    })
```